src/main/java/hudson/plugins/logparser/LogParserPublisher.java

```diff
@@ -106,11 +106,17 @@ public void setUseProjectRule(final JSONObject useProjectRule) {
 
     @DataBoundSetter
     public void setParsingRulesPath(final String parsingRulesPath) {
+        if (parsingRulesPath != null) {
```

Comment:
this is to simplify the workflow notation (didn't know how to pass the `JSONObject` to `setUseProjectRule`)